lib/styles/react-joyride.scss

```diff
@@ -157,7 +157,7 @@ $joyride-tooltip-arrow-scale: ($joyride-tooltip-arrow-size / ($joyride-tooltip-a
     &.bottom-right {
       margin-top: $joyride-tooltip-arrow-height;
 
-      .triangle {
+      .joyride-tooltip__triangle {
```

Comment:
Selector `joyride-tooltip__triangle` should be written in lowercase with hyphens